integration_cdf_main.py

```diff
@@ -0,0 +1,30 @@
+#%%
+import json
+from tkinter import filedialog
+
+from tomni.cytosmart_data_format import CytoSmartDataFormat
+
+#%%
+json_fp = filedialog.askopenfilename(title="Select CDF JSONs.")
+
+with open(json_fp, "rb") as f:
+    dicts = json.load(f)
+cdf = CytoSmartDataFormat.from_dicts(dicts=dicts)
+
+#%%
+print(f"__len__: {len(cdf)}")
+
+#%%
+# cdf creates a generator.
+count = 0
+for annotation in cdf:
+    count += 1
+print(f"Count: {count}")
+
```

Comment:
```suggestion
print(f"Dict count:{len(dicts)}")
```
to show that they are equal in length